### ISSUE TITLE
How do I know if a color scheme is continuous or categorical?

### ISSUE THREAD

**Issue by eliascarv** [JuliaGraphics/ColorSchemes.jl#118]

**Comment by cormullion** [comment]
Aren’t all colorschemes categorical (having `n` colors in `.colors`) and continuous (with `get(scheme, f)`)? 

Or I don’t understand the question yet! 😀

**Comment by eliascarv** [comment]
You're absolutely right @cormullion, it doesn't make sense to think about categorical or continuous colorscheme, since it can be both. Thanks for the answer!
